index.js

```diff
@@ -1,7 +1,100 @@
 // Iteration 1: Names and Input
 
+const hacker1 = "Juan";
+const hacker2 = "Alberto";
 
-// Iteration 2: Conditionals
+console.log(`The driver is ${hacker1}`);
+console.log(`The navigator is ${hacker2}`);
 
+// Iteration 2: Conditionals
+let longest;
+if ( hacker1.length > hacker2.length) {
+    longest = hacker1;
+    console.log(`The driver has the longest name, it has ${longest.length} characters.`)
+} else if ( hacker2.length > hacker1.length) {
+    longest = hacker2;
+    console.log(`It seems that the navigator has the longest name, it has ${longest.length} characters.`)
+} else if ( hacker2.length === hacker1.length) {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
```

Comment:
Entiendo el porque del longest!
Pero no te haría falta, si usas directamente el elemento asi te ahorras una variable! :D

```suggestion
if ( hacker1.length > hacker2.length) {
    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
} else if ( hacker2.length > hacker1.length) {
    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
} else {
    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
}
```